src/components/ListItem/ListItem.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './ListItem.css';
+import Item from '../../components/Item/Item';
+var Columns = require('react-columns');
+function ListItem() {
```

Comment:
The component name could be something like DesignList. The Item part of the name is little confusing as this is not an list item but list itself.